<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="TetrisRenderer" name="board">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>200</width>
      <height>400</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>191</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Shapes</string>
    </property>
    <widget class="QLabel" name="labelZShapeText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>46</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ZShape:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelZShape">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>21</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSShapeText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>46</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>SShape:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSShape">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelLineShapeText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>LineShape:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelLineShape">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTShapeText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>46</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TShape:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTShape">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSquareShapeText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>SquareShape:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSquareShape">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelLShapeText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>46</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>LShape:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelLShape">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMLShapeText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>MLShape:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMLShape">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelSummeShapesText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Summe:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSummeShapes">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Scores</string>
    </property>
    <widget class="QLabel" name="labelLevel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="LabelLinesRemoved">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelScore">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelLevelText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>46</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Level:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelScoreText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>46</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Score:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelLinesRemovedText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Lines Removed:</string>
     </property>
    </widget>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>160</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>371</width>
      <height>271</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>90</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>20</number>
    </attribute>
    <row>
     <property name="text">
      <string>Height</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Holes</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Blockades</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>ColumnSpace</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Touchfloor</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Touchwall</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Touchpiece</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>RowSpace</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Blockcount</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>LinesRemoved</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Well</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>LandingHeight</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Weight</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Count</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="9" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="9" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="9" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="10" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="10" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="10" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="11" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="11" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="11" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="checkBoxDrop">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>320</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Drop</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxFPS">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Limit Fps</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelSpeed">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>420</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxNoOutput">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>No Output</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelIndividuumText">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>310</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Individuen Evaluiert:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelIndividuum">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>310</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>912</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TetrisRenderer</class>
   <extends>QFrame</extends>
   <header>tetrisrenderer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
